charts/darts-portal/values.dev.template.yaml

```diff
@@ -5,3 +5,4 @@ nodejs:
   environment:
     DARTS_API_URL: https://darts-api.staging.platform.hmcts.net
     DARTS_PORTAL_URL: https://${SERVICE_FQDN}
+    MANUAL_DELETION_ENABLED: true
```

Comment:
I think we need to remove this, we want to default it to false.

Reply:
We have E2E tests which will fail for file-deletion pages. we need to modify these for dev/staging env. 
I think we can disable this for prod but keep enable this for lower env. Please advise.